src/main/java/com/deusgmbh/xcusatio/data/usersettings/UserSettings.java

```diff
@@ -16,18 +16,20 @@
     private Sex sex;
     private Address home;
     private Address university;
-    private QuickSettings quickSettings;
+    private ExcusesVibesMode excusesVibesMode;
+    private ExcusesVibes excusesVibes;
     private Set<String> interests;
 
     public UserSettings(String googleCalendar, int age, Sex sex, Address home, Address university,

```

Comment:
für Kalender brauchen wir zumindest später nen json und nen refresh token (json) beides im Profil hinterlegt. Schreibs vlt. als To-Do auf, damit wir dran denken.